assets/styles.css

```diff
@@ -0,0 +1,765 @@
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@200;300;400;500;600;700;800;900&display=swap');
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #f5f5f5;
+  font-family: 'Mulish', sans-serif;
+}
+
+ul {
+  list-style-type: none;
+  margin-left: -40px;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.flex {
+  display: flex;
+}
+
+.grid {
+  display: grid;
+}
+
+.container,
+nav,
+.story .banner,
+.story .text,
+.category,
+.featured-articles,
+.featured-article::after,
+.deal,
+.deal-img,
+.category .content::before,
+.card-header,
+.logo-date,
+.copyright-text {
+  width: 100%;
+}
```

Comment:
- [ ] [OPTIONAL] You should apply the same "common-class" approach suggested by the previous TSEs here (and other places too) so your CSS file has a lot less lines.